### ISSUE TITLE
Different APK hash on fdroid

### ISSUE THREAD

**Issue by tbshfr** [syncthing/syncthing-android#1902]
Version 1.23.1 released on fdroid doesn't match with the release on github.

Github APK ([sha256sum.txt.asc](https://github.com/syncthing/syncthing-android/releases/download/1.23.1/sha256sum.txt.asc)):
e032b2ce1f6979c81a950f92bb11a34a29c4a8ecdd09159370b1800477ca31e0

Fdroid APK:
aceb5b5dfa46aa86ba2a07cddef9796f3359b1c207ba0009394c95986727e427

I expected the same hash on both platforms. 
If this is expected, please add the fdroid hash to the release notes or post them in the forum.

### Version Information

    App Version: 1.23.1
    Syncthing Version: not relevant
    Android Version: not relevant, same when downloading from fdroid website on PC.

**Comment by tomasz1986** [comment]
This is normal because F-Droid compiles their releases with a different signature.

**Comment by tbshfr** [comment]
I think if the builds arent reproducible, the hash should be published.
The syncthing [website](https://syncthing.net/downloads/) 
 lists fdroid as a download option, so I have to assume it is not a third party distribution. Also on the distribution channel fdroid isn't listed as 3rd party repo. I think that it would be appropriate to publish the apk hash somewhere.

**Comment by imsodin** [comment]
tl;dr: No.

F-Droid is a separate, great project we link to as it gives our users a means to install this app from an open source based ecosystem, for those that care and are able to circumvent googles walled garden. They of course do publish signatures for their own builds, which they display prominently right next to the apk download link: https://f-droid.org/en/packages/com.nutomic.syncthingandroid/
